src/components/MetaTextBlock/index.js

```diff
@@ -0,0 +1,49 @@
+import { RichTypography } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import Section from "@/pesayetu/components/Section";
+
+const TwoColumnLayout = ({ content, title, subtitle, ...props }) => {
+  const classes = useStyles(props);
+  if (!content) {
+    return null;
+  }
+
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <Grid justifyContent="space-between" container>
+          <Grid item xs={12} lg={4}>
+            <Typography className={classes.title} variant="h1">
```

Comment:
```suggestion
            <Typography className={classes.title} variant="h2">
```

https://github.com/CodeForAfrica/PesaYetu/blob/b0c9a40839e9df6e0ca1cea46906420457d32794/src/theme/index.js#L127-L131